hackaton.iml

```diff
@@ -5,7 +5,6 @@
     <content url="file://$MODULE_DIR$">
```

Comment:
I znów plik iml ;) Czyżbym ja go kiedyś niechcący wrzucił (zostawił) za waszymi namowani? ;)

Reply:
Sam stwierdziłeś że .idead nie może być ale iml ujdzie :P

Reply:
Bo zaczęliście krzyczeć, że to muszą być te rzeczy ;) Ale to i tak szczegół mały z którego będziemy się kiedyś śmiać 🗡